projects/ad469x_fmc/common/ad4696_bd.tcl

```diff
@@ -0,0 +1,147 @@
+
+create_bd_intf_port -mode Master -vlnv analog.com:interface:spi_master_rtl:1.0 ad4696_spi
+create_bd_port -dir O ad4696_spi_cnv
+create_bd_port -dir I ad4696_spi_busy
+
+## To support the 1MSPS (SCLK == 80 MHz), set the spi clock to 160 MHz
+
+ad_ip_instance axi_clkgen spi_clkgen
+ad_ip_parameter spi_clkgen CONFIG.CLK0_DIV 5
+ad_ip_parameter spi_clkgen CONFIG.VCO_DIV 1
+ad_ip_parameter spi_clkgen CONFIG.VCO_MUL 8
+ad_connect $sys_cpu_clk spi_clkgen/clk
+ad_connect spi_clk spi_clkgen/clk_0
+
+# create a SPI Engine architecture
+
+create_bd_cell -type hier spi_ad4696
```

Comment:
Why do we not use the SPI Engine process to create a SPI instance?

Reply:
initially it was simply ported. Project has been updated